src/Core/Pages/Send/SendAddEditPage.xaml.cs

```diff
@@ -181,6 +181,17 @@ private async void FileType_Clicked(object sender, EventArgs eventArgs)
 
         private void OnMaxAccessCountTextChanged(object sender, TextChangedEventArgs e)
         {
+            var maxAccessEntry = (Microsoft.Maui.Controls.Entry)sender;
+
+#if IOS
+            //Workaround: To avoid a bug that incorrectly sets the TextColor when changing text
+            // programatically we need to set it to null and back to the correct color
+            // MAUI issue https://github.com/dotnet/maui/pull/20100
+            maxAccessEntry.TextColor = null;
+            var color = ThemeManager.GetResourceColor("TextColor");
+            maxAccessEntry.TextColor = color;
+#endif
```

Comment:
🎨 No need for the extra variable, and `HACK` might be better given that it's tracked by Tasks:
```suggestion
#if IOS
            // HACK: To avoid a bug that incorrectly sets the TextColor when changing text
            // programatically we need to set it to null and back to the correct color
            // MAUI issue https://github.com/dotnet/maui/pull/20100
            maxAccessEntry.TextColor = null;
            maxAccessEntry.TextColor = ThemeManager.GetResourceColor("TextColor");
#endif
```